src/gravel/controllers/nodes/mgr.py

```diff
@@ -344,6 +347,53 @@ async def finish_bootstrap_cb(
             logger.error(f"bootstrap error: {e.message}")
             raise NodeCantBootstrapError(e.message)
 
+    def _spawn_etcd(self, new: bool, token: Optional[str]) -> None:
+
+        assert self._state.hostname
+        assert self._state.address
+        hostname = self._state.hostname
+        address = self._state.address
+
+        logger.info(f"starting etcd, hostname: {hostname}, addr: {address}")
+
+        client_url: str = f"http://{address}:2379"
+        peer_url: str = f"http://{address}:2380"
+        args_dict: Dict[str, str] = {
+            "name": hostname,
+            "initial-advertise-peer-urls": peer_url,
+            "listen-peer-urls": peer_url,
+            "listen-client-urls": f"{client_url},http://127.0.0.1:2379",
+            "advertise-client-urls": client_url,
+            "initial-cluster": f"{hostname}={peer_url}",
```

Comment:
So the name of the etcd cluster is the hostname of the first node. Would it make sense to use `"initial-cluster": f"aquarium={peer_url}"` instead?